nsxt/data_source_nsxt_management_cluster.go

```diff
@@ -0,0 +1,74 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceNsxtManagementCluster() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNsxtManagementClusterRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Description: "Unique identifier of this cluster.",
+				Optional:    true,
```

Comment:
If this is computed-only attribute, we should remove "Optional: true" (same below). In this case terraform will not allow setting these in tf.